controllers/metal3machine_controller.go

```diff
@@ -345,9 +346,10 @@ func (r *Metal3MachineReconciler) Metal3ClusterToMetal3Machines(o handler.MapObj
 // BareMetalHost and that BareMetalHost references a Metal3Machine.
 func (r *Metal3MachineReconciler) BareMetalHostToMetal3Machines(obj handler.MapObject) []ctrl.Request {
 	if host, ok := obj.Object.(*bmh.BareMetalHost); ok {
+		capm3GV := schema.ParseGroupResource(capm3.GroupVersion.String())
```

Comment:
isn't capm3.GroupVersion already capm3GV ?

Reply:
You are right! I fixed it!